etl/controller/pulsemaintainer.go

```diff
@@ -65,14 +66,14 @@ func (c *Controller) pulseSequence(ctx context.Context) {
 			log.WithField("sequential_pulse", c.sequentialPulse)
 
 			nextSequential, err = c.storage.GetPulseByPrev(c.sequentialPulse)
-			if err != nil {
+			if err != nil && !gorm.IsRecordNotFoundError(err) {
 				log.Errorf("During loading next sequential pulse: %s", err.Error())
 				return
 			}
 
 			if nextSequential == emptyPulse {
 				toPulse, err := c.storage.GetNextSavedPulse(c.sequentialPulse)
```

Comment:
if there is not next saved pulse, we can log `Info` message about it and `return`, we can't reload data because we don't now to what pulse we should reload. And that way changes at `reloadData` func not needed.

Reply:
you right, it works.